### ISSUE TITLE
Hardware Survey NEVER shows up in Big Picture

### ISSUE THREAD

**Issue by adolson** [ValveSoftware/steam-for-linux#4289]
However, I've never seen a survey pop up in Big Picture mode, and I've never seen a confirmation from anyone else that it does, including on SteamOS. I _have_ had the Steam Hardware Survey several times since I began using Steam, but only ever in desktop mode. Therefore, SteamOS and other systems set to boot directly into Big Picture will never be included in these stats.

Which wouldn't be such a big deal, if it weren't for the fact that the Hardware Survey is being used by developers to point out how low Linux's market share is as a reason to avoid porting to SteamOS/Linux. ( Just one example: https://twitter.com/TheRevillsGames/status/694422282333065216 )

**Comment by gdrewb-valve** [comment]
I don't believe this is intended to work on any platform (Big Picture just doesn't do surveying), but leaving this open in case that's incorrect.

**Comment by NaughtySquid** [comment]
I know we're not supposed to randomly comment on bug reports, but surly this is a genuine issue. I mean, how will SteamOS ever be surveyed? Makes the Steam Hardware Survey results never show SteamOS?

**Comment by gdrewb-valve** [comment]
I meant "not intended to work" in the sense that the work hasn't been done to hook it up so this isn't a bug where code isn't working as intended.  Having survey information would be valuable and eventually that'll get added.

**Comment by NaughtySquid** [comment]
I see, thanks for letting us know.

**Comment by adolson** [comment]
This is getting out of hand.
http://www.pcworld.com/article/3040719/linux/linux-usage-on-steam-continues-to-fall.html

> **Steam machines haven’t stopped the slide**
> The evidence comes from Valve’s Steam Hardware Survey. It reveals that machines running Linux—including SteamOS—were at 0.91 percent in February 2016, a drop of 0.04 percent from January.

I wish the "hardware" survey didn't even report the OS at all. This is doing more harm than good, as it seems absolutely nobody with a mouth or a dev team realizes the limitations of the OS portion of the survey.

**Comment by Translator5** [comment]
Wooohooo -> Just updated to nVidia 370 Beta-Driver and the Survey showed up the firist time :)

**Comment by adolson** [comment]
> Wooohooo -> Just updated to nVidia 370 Beta-Driver and the Survey showed up the firist time :)

In Big Picture???

**Comment by Translator5** [comment]
Ou sorry, false topic. Nope just in Normal UI.

**Comment by Managor** [comment]
This issue is likely outdate and can be closed
